inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -246,3 +246,16 @@ persistent_volumes_enabled: false
 ## See https://github.com/kubernetes-incubator/kubespray/issues/2141
 ## Set this variable to true to get rid of this issue
 volume_cross_zone_attachment: false
+
+## Container Engine Acceleration
+## Enable container accelartion feature, for example use gpu acceleration in containers
+# nvidia_accelerator_enabled: true
+## Nvidia GPU driver install. Install will by done by a (init) pod running as a daemonset.
+## Array with nvida_gpu_nodes, leave empty or comment if you dont't want to install drivers.
+## Nodes won't get labels gpu labels if they are not in the array.
+## Important: this should be set in all.yml 'docker_storage_options: -s overlay2'
+# nvidia_gpu_nodes:
+#   - kube-gpu-001
+# nvidia_driver_version: "384.111"
+## flavor can be tesla or gtx
```

Comment:
Is it possible to have a mix of both across the fleet (though maybe 1 type per node?)

Reply:
NVM, I believe GTX cards work with TELSA drivers in practice.